src/App.tsx

```diff
@@ -1,41 +1,98 @@
-import React from 'react';
-import './App.scss';
-import './styles/general.scss';
+import React, { useEffect, useState } from 'react';
 import { PostsList } from './components/PostsList';
 import { PostDetails } from './components/PostDetails';
+import { getUsers } from './api/api';
+import { getUserPosts } from './api/posts';
+
+import './App.scss';
+import './styles/general.scss';
+
+const App: React.FC = () => {
+  const [users, setUsers] = useState<User[] | null>([]);
+  const [currentUserId, setCurrentUserId] = useState(0);
+  const [selectedPostId, setSelectedPostId] = useState(0);
+  const [posts, setPosts] = useState<Post[] | null>([]);
+
+  const getUsersFromServer = async () => {
+    try {
+      const usersFromServer = await getUsers();
+
+      setUsers(usersFromServer);
+    } catch {
+      setUsers([]);
+    }
```

Comment:
it would be great to show user some message about error instead of setting empty array

Reply:
Not sure about this) couldn't find the way to simulate the situation with error happening